src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,45 @@
+package controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import service.UserJoinService;
+import util.HttpRequest;
+import util.HttpResponse;
+
+public class UserController {
+    private final String HTTP_GET = "GET";
+    private final String JOIN_FORM = "/user/form.html";
+    private final String CREATE_USER_URL = "/user/create";
+    private UserJoinService userJoinService;
+    private Logger log = LoggerFactory.getLogger(getClass());
+
+    public UserController(UserJoinService userJoinService) {
+        this.userJoinService = userJoinService;
+    }
+
+    // TODO : 에러페이지 생성, 회원가입 검증
+    public String process(HttpRequest request, HttpResponse response) {
+        // GET 요청인 경우 분리
+        if (request.getMethod().equals(HTTP_GET)) {
+            // 회원 가입 폼 보여주기
+            if (request.getUrl().equals(JOIN_FORM)) {
+                return "/user/form.html";
+            }
```

Comment:
```suggestion
            if (request.getUrl().equals(JOIN_FORM)) {
                return JOIN_FORM;
            }
```
두 개 값이 같아서 이 코드와 같은 의미입니다.

Reply:
이 부분을 없애고 범용적인 메서드로 따로 빼서 리팩토링할 수 있을 것 같아요.